website/content/docs/connect/registration/sidecar-service.mdx

```diff
@@ -50,7 +50,7 @@ definitions:
 {
   "service": {
     "name": "web",
-    "port": 8080,
+    "port": 8080
   }
 }
```

Comment:
Based on the proceeding text, I think it would be more clear to separate these registrations into separate code blocks. For example:

```json
{
  "service": {
    "name": "web",
    "port": 8080
  }
}
```

```json
{
  "name": "web-sidecar-proxy",
  "port": 20000,
  "kind": "connect-proxy",
  "checks": [
    {
      "Name": "Connect Sidecar Listening",
      "TCP": "127.0.0.1:20000",
      "Interval": "10s"
    },
    {
      "name": "Connect Sidecar Aliasing web",
      "alias_service": "web"
    }
  ],
  "proxy": {
    "destination_service_name": "web",
    "destination_service_id": "web",
    "local_service_address": "127.0.0.1",
    "local_service_port": 8080,
  }
}
```

Reply:
Better yet, this could be left as a single JSON object that is placed in the agent's config file and registers both services.

```json
{
  "services": [
    {
      "name": "web",
      "port": 8080
    },
    {
      "name": "web-sidecar-proxy",
      "port": 20000,
      "kind": "connect-proxy",
      "checks": [
        {
          "Name": "Connect Sidecar Listening",
          "TCP": "127.0.0.1:20000",
          "Interval": "10s"
        },
        {
          "name": "Connect Sidecar Aliasing web",
          "alias_service": "web"
        }
      ],
      "proxy": {
        "destination_service_name": "web",
        "destination_service_id": "web",
        "local_service_address": "127.0.0.1",
        "local_service_port": 8080,
      }
    }
  ]
}
```

@krastin Are you open to making this change to your PR?

I'd will gladly approve it if you modify the PR to accept either suggestion.